### PR TITLE
Added html and css codes. Added all the sections.

### PR DESCRIPTION
**Pull request by daviidy**
Worked on all the requirements given on this project:

- Used semantic HTML elements for the header, the main content, the form and the footer.
- Use Flexbox and Grid to align the elements.
- Use Grid for The Theory of Inflation section
- Styled the elements to look like the real webpage.
- Fixed linters errors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,854 @@
...
+  <header class="menu-container">
+    <section class="menu">
+
+      <div class="left-menu">
+        <i class="fas fa-bars"></i>
+        <i class="fas fa-search"></i>
+        <span>Space & Cosmos</span>
+      </div>
+      <div class="center-menu">
+        <img src="./images/logo.png" alt="New York Times Logo">
```
Comment (acushlakoncept) [inline index.html]: The NY Times logo should be a link

**Review by acushlakoncept — CHANGES_REQUESTED** [review]
# Status: Changes Required ♻️

Hello @daviidy and @sinansevgi, Great job! 👏  Both of you have done well on this project.
Let's make a few changes, and we will be done.

- [x] Sticky bottom footer. The fixed position footer that is on the original page is required in this project.

- [x] There are errors in the [w3c validator](https://validator.w3.org/).

- [x]  A combination of all three techniques ([float](https://css-tricks.com/all-about-floats/), [flex](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) and [grid](https://css-tricks.com/snippets/css/complete-guide-grid/)) is required in this project. You have only used flex and grid. Implementing float should not be a big deal for you. 
flex_float_grid_requirement_pathwright

- [x] In **The Theory of Inflation section**, try using the sprite image as it is on the original page without cutting them, [see image](https://static01.nyt.com/newsgraphics/2014/03/18/theory-of-inflation/1ebaa1b972edda0f407fbaa0e010f6b50fb6792e/inflation-945.png)

### Cheers and Happy coding! 💻 👏

(Feel free to contact me on slack(@acushla), if you have any additional question)
#### Uduak Essien
[Twitter](https://twitter.com/acushlakoncept) | [LinkedIn](https://www.linkedin.com/in/acushlakoncept/)

**Comment by umairarshadbutt** [comment]
##
## Pull Request under review by Umair Arshad (TSE)

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @daviidy AND @sinansevgi 
Good job with the changes! :tada: We are almost there.
We, however, do have some few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly make sure the changes as previous TSE suggested.
- [x] Fix HTML validation [HTML Validation Screenshort](https://user-images.githubusercontent.com/29689273/93863699-42fbed80-fcdd-11ea-985c-1aa7caaeaf8d.png). You can check HTML validations in [validator.w3.org](https://validator.w3.org/)


##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you stick at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by arslanbisharat — APPROVED** [review]
### Congratulations APPROVED :clap:
Hello I can see you worked really hard on this project. and I'm very happy to approve this project with the latest changes you made in this project. :satisfied:
_The clone shows a good understanding and usage of HTML CSS💯_

- The cloned page is rendering as it should and meets all the project specifications.
-  CSS is clean.
-  The markup as well has no errors nor warnings to show. Good job with that.
-  The repo has followed all the guidelines there are to follow.
-  All the required tasks have been fulfilled in this project.
-  Congratulations on the splendid job👍
Now you can merge the branches and move forward with your curriculum. 🎆 

[OPTIONAL] If you wish to delete your currently working branch please update the Live demo link too so that it works properly. 

_Keep working hard._ 

I am very excited to see your next project.

**Happy Coding**:smiley:
------------------------------------------------------------------------------
**Regards,
Arslan Bisharat**
[linkedin](https://www.linkedin.com/in/arslanbisharat/) slack: @arslanbisharat

**Pull request merged**